### PR TITLE
Better squashing for FlowPathCount and FlowRunCount

### PR DESCRIPTION
**Pull request by rowanseymour**
Adds a `is_squashed` field so that we can be certain which rows have been squashed.

Implemented so far for the squashables in the flows app - wanted to check how folks feel about this before implementing for the others.

We currently squash `FlowPathCount` instances every 5 minutes but sometimes the squash process takes longer than 5 minutes, so would be good to fix that one sooner than later.

### CODE REVIEW

File: temba/sql/0086_flows.sql [hunk 1]
```diff
@@ -0,0 +1,263 @@
...
+   ON flows_flowstep
+   FOR EACH ROW
+   EXECUTE PROCEDURE temba_update_flowpathcount();
+
+-- Install TRUNCATE trigger
+DROP TRIGGER IF EXISTS temba_flowstep_truncate_flowpathcount on flows_flowstep;
+CREATE TRIGGER temba_flowstep_truncate_flowpathcount
+  AFTER TRUNCATE
+  ON flows_flowstep
+  EXECUTE PROCEDURE temba_update_flowpathcount();
```
Comment (ericnewcomer) [inline temba/sql/0086_flows.sql]: Maybe fix this while we are at it? Shouldn't add rows to mark the deleted steps. Should just delete the related counts for these steps.

Comment (rowanseymour) [inline temba/sql/0086_flows.sql]: Not getting what is wrong with how it currently is. If we delete a step, we need to decrement the count for the flow path segment that step is on.. no?

Comment (ericnewcomer) [inline temba/sql/0086_flows.sql]: Oh, ya, mispoke. I think it's when flows are deleted, cascading to their steps, we then try to add negative counts which then blow up.

Comment (rowanseymour) [inline temba/sql/0086_flows.sql]: Ah yeah that's the weird case that i ran into in the unit tests recently - thankfully we don't delete flows in real life

**Pull request merged**